### ISSUE TITLE
Parse "custom" xbrl files

### ISSUE THREAD

**Issue by jakobdo** [greedo/python-xbrl#28]
I'm unsure if this is a bug or just my who is to stupid to get this lib working.
I have tried to work with danish xbrl files from the danish site: virk.dk
Look at this f.ex:
https://datacvr.virk.dk/data/visenhed?enhedstype=virksomhed&id=27400744&soeg=danmark&type=Alle
Under "regnskaber" you should find a link to xbrl-files.
F.ex. this: https://datacvr.virk.dk/data/offentliggorelse?dl_ref=Y3ZyLmRrOi8veGJybHMvWC03RENCRUIxMi0yMDE1MDEzMF8xNjA5NTJfNzA0
When i look into the file, the xml is in this format:

I see tags like: e:ProfitLoss
But when i look into your parser, it looks for:
income_loss += xbrl.find_all(name=re.compile("(us-gaap:profitloss)", re.IGNORECASE | re.MULTILINE))

So is the only xbrl files supported, the files with: us-gaap: infront of the "tags" ?

**Comment by heisen273** [comment]
Any updates on this issue? I'd love to parse virk.dk

**Comment by greedo** [comment]
Yes @heisen273 currently US GAAP is the only accounting standard supported, through I could easily add support for other accounting standards

**Comment by heisen273** [comment]
@greedo Cmon, mate, dont hesitate to do it. I can help you with this as hard as I can, I've spent some time with your lib and modified it for my needs, - lemme know if there's anything i can help you with.

**Comment by jakobdo** [comment]
A dynamic "identifier" would be nice.
Or if you could create a schema or similar, then it would be "easy" to implement new standards. (i think)
